### ISSUE TITLE
Java 2.1: Byte Code

### ISSUE THREAD

**Issue by garvitraj** [girlscript/winter-of-contributing#1196]
<hr>

## Description 📜

Add audio, documentation, and video regarding the **Bytecode**.

<hr>

## Domain of Contribution 📊

- [x] Java

<hr>

## Location of File to be added

The files should be added inside the `Bytecode` folder which is inside the `Fundamental` present in the  `Java` main folder.  The documentation file can be in the `.md`/`.py`/`.ipynb` format. Audio should be in `.mp3` format and to be submitted through a Google drive link. Video can be in any format and should be submitted through a Google Drive link.

 **Note:**
 - You are required to comment with `\assign CONTENT_TYPE`, so that other content types can be created at the same time.
 - Failing to comment in the above manner will lead to the removal of assignment from the issue even if the bot assigns.
 - You are required to do a PR within 7 days otherwise you will be unassigned and others will be assigned.
 - Issues will be assigned on a first come first serve basis.

**Comment by Amrita-Mukherjee** [comment]
\assign Document

**Comment by Gagarod** [comment]
\assign documentation

**Comment by garvitraj** [comment]
> \assign Document

@AbhinandanAdhikari  go with the Documentation

**Comment by garvitraj** [comment]
> \assign Audio

@Binita-tech go with the audio

**Comment by garvitraj** [comment]
@Binita-tech pls go with one issue at a time

**Comment by Binita-tech** [comment]
Ok

**Comment by Pranav120602** [comment]
\assign Video

**Comment by AkMo3** [comment]
Slot for audio is empty. Documentation will be done by @Amrita-Mukherjee and video will done @Pranav120602.

**Comment by AbhinandanAdhikari** [comment]
> Slot for audio is empty. Documentation will be done by @Amrita-Mukherjee and video will done @Pranav120602.

@AkMo3 I've been assigned for documentation

**Comment by AkMo3** [comment]
> > Slot for audio is empty. Documentation will be done by @Amrita-Mukherjee and video will done @Pranav120602.
> 
> @AkMo3 I've been assigned for documentation

Yeah, my bad. I meant for you only. @AbhinandanAdhikari has been appointed for documentation.

**Comment by Minal5** [comment]
\assign audio

**Comment by jaigora24** [comment]
\assign document
